torch/csrc/api/include/torch/nn/functional/activation.h

```diff
@@ -161,9 +160,8 @@ inline Tensor softmin(const Tensor& input, int64_t dim,
 }
 } // namespace detail
 
-inline Tensor softmin(const Tensor& input, SoftminFuncOptions options,
-                      c10::optional<torch::Dtype> dtype = c10::nullopt) {
-  return detail::softmin(input, options.dim(), dtype);
+inline Tensor softmin(const Tensor& input, SoftminFuncOptions options) {
```

Comment:
functional.py has default values for all options in:
`def softmin(input, dim=None, _stacklevel=3, dtype=None)`
so maybe
```suggestion
inline Tensor softmin(const Tensor& input, SoftminFuncOptions options = {}) {
```

Reply:
replied in an earlier comment